src/app/infrastructure/chooseQuestion.js

```diff
@@ -0,0 +1,12 @@
+import { async } from 'regenerator-runtime'
+import {RandData} from './RandData'
+
+export const chooseQuestions = async (num) => {
+    const inputData = await RandData.randData()
+    const question = {
```

Comment:
To co zwraca funkcja lepiej zamknąć w klasie i tą klasę dodać do folderu `/app/domain/`.